.vscode/settings.json

```diff
@@ -0,0 +1,12 @@
+{
+    "files.exclude": {
+        "**/__pycache__": true,
```

Comment:
perhaps we want to update the gitignore file so that we ignore everything except the tasks and launch files?